website/docs/server_sdk.md

```diff
@@ -11,7 +11,115 @@ This page documents how to use UPS
  - AEROGEAR-10139	
 ### Node Sender
  - AEROGEAR-10138
-### REST Sender
- - AEROGEAR-10137	
+
+### REST Sender 
+- AEROGEAR-10137
+
+The _UnifiedPush Server_ supports a RESTful endpoint found at `/rest/send`. For connecting to RESTful clients this endpoint consumes and produces _JSON_ serialised messages that are protected by HTTP Basic authentication, returning a _202 Accepted_ status for message acceptance and a _401 Unauthorized_ status when there is an authentication error between the server and application.
+
+#### Authentication
+The UPS RESTful endpoint receives a HTTP POST request and checks the HTTP Basic header to extract the  pushApplicationId and the masterSecret which make up a token in custom header and are sent as part of the application message for authentication against the push client.
+
+This can be seen in an example curl request 
+    
+      ```bash
+      curl -u "PushApplicationID:MasterSecret"
+        -v -H "Accept: application/json" -H "Content-type: application/json"
+        -X POST
+        -d '{
+          "message": {
+           "alert": "HELLO!",
+           "sound": "default",
+           "user-data": {
+               "key": "value",
+           }
+        }'
+        https://SERVER:PORT/CONTEXT/rest/sender
+        ```
+     
+
+#### Data Format 
+The UnifiedPush message sent is Json formatted instances of `org.jboss.aerogear.unifiedpush.message`. This message is made up of three different collections of information:
+
+ - The Message 
```

Comment:
I would like to see some details for fields from this doc https://aerogear.org/docs/unifiedpush/push-message-format/  you can remove the references to windows and simple-push because those are no longer used.